common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -463,4 +463,15 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  // Owner: Maria Livia Chiorean
+  val SharingComments = Switch(
+    SwitchGroup.Feature,
+    "sharing-comments",
+    "When ON, the user will be able to share comments",
+    owners = Seq(Owner.withGithub("marialivia16")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
This discussion may have already happened, but how will we determine this is a good thing? How can we test this? Should we maybe put in a `sellByDate` as a reminder to review this feature?